WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -89,12 +89,19 @@ __Fact__: code review is one of the few scientifically proven code improvement t
    [alias]
       upd = "!f(){ git fetch upstream && git checkout master && git rebase upstream/master && git push -f origin master && git branch --merged; };f"
       del = "!f(){ git checkout master && (git branch -D \"$1\"; git push origin --delete \"$1\"); git branch; };f"
-      pr = "!f(){ git checkout -b $(date +'%s') && git add . && git commit -S -m \"$1\" && git push -u origin $(git branch --show) && start \"$(echo $(git config --get remote.upstream.url) | sed 's/.git$//g')/compare/master...username:$(git branch --show)?expand=1\"; };f"
+      pr = "!f(){ git checkout -b $(date +'%s') && git add . && git commit -S -m \"$1\" && git push -u origin $(git branch --show) && start \"$(echo $(git config --get remote.upstream.url) | sed 's/.git$//g')/compare/master...$(git config --get user.name):$(git branch --show)?expand=1\"; };f"
    ```
   You can use it with the console in the project library by the following commands:  
    - `git upd` -> Update the forked repository  
    - `git del` -> Delete the current branch remotely and locally  
    - `git pr "Commit message"` -> Create a new branch, commit then push your work, open your browser with a new PR waiting for you to fill the title and the description.
+Make sure to have these lines -
+```
+[user]
+	name = your_github_username
+	email = your_github@email.com
+```
+in the .gitconfig file
```

Comment:
I would move this line up:
`Make sure to have these lines in the .gitconfig file`